app/src/main/java/ch/epfl/sdp/peakar/social/SocialActivity.java

```diff
@@ -2,43 +2,106 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.net.Uri;
 import android.os.Bundle;
+import android.view.View;
 import android.widget.ListView;
 
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 import ch.epfl.sdp.peakar.R;
+import ch.epfl.sdp.peakar.user.friends.FriendItem;
+import ch.epfl.sdp.peakar.user.services.AuthService;
 import ch.epfl.sdp.peakar.utils.MenuBarHandler;
 
 import static ch.epfl.sdp.peakar.utils.StatusBarHandler.StatusBarLightGrey;
 import static ch.epfl.sdp.peakar.utils.TopBarHandler.setupGreyTopBar;
+import static ch.epfl.sdp.peakar.utils.TopBarHandler.setupSwitch;
 
 public class SocialActivity extends AppCompatActivity {
 
     private final static List<SocialItem> globalSocialItems = Collections.synchronizedList(new ArrayList<>());
```

Comment:
Maybe we can also add here a friendSocialItems list?

Reply:
I didn't do it since it is not synced like the global. Therefore it is only used once when creating the list adapter. It will be static like the global later. Is that ok?

Reply:
Oh, you're right, it's fine then!